index.bs

```diff
@@ -7,3 +7,514 @@ URL: https://netwerk-digitaal-erfgoed.github.io/requirements-network-of-terms/
+--------------------
+**Priority**: Should have
+
+The Network of Terms has an endpoint for querying terminology sources (see [requirement 1](#1)).
+However, an endpoint is meant to be used by the systems of users, not directly by users.
+Yet users want to be able to query the Network of Terms to get an impression of the possibilities and the results,
+regardless of how the users' systems use it.
+The Network of Terms therefore has an online search environment for users.
+An example of such an environment is the [demonstrator](https://termennetwerk.netwerkdigitaalerfgoed.nl/) of the Network of Terms.
+
+The Network of Terms searches terminology sources registered in the Registry {#3}
+--------------------
+**Priority**: Must have
+
+The Network of Terms searches terminology sources, but does not determine which sources can be used.
+For this purpose the Network of Terms relies on the Registry. The Registry is the generic facility of the Dutch Digital Heritage Network
+that keeps track of the datasets in the network, including the available terminology sources.
+Use of the Registry improves clarity: anyone — both man and machine — who wants to do anything with datasets can use the Registry as an entry point.
+
+Use of the Registry does not imply that the Network of Terms is obliged to make all terminology sources in the Registry searchable.
+The Network of Terms can choose to use a selection of sources, depending on its purpose and target audience.
+This may be the case, for example, with distributed instances of the Network of Terms.
+For example: instance A searches only terminology sources used in the archives sector and instance B only searches terminology sources in the museums sector.
+
+The Network of Terms publishes information about the terminology sources it can search via an endpoint {#4}
+--------------------
+**Priority**: Must have
+
+The Network of Terms makes clear to users which terminology sources it can query.
+Although users can consult the Registry to see what terminology sources are available (see [requirement 3](#3)),
+the Registry is not aware of the sources used by the Network of Terms.
+For example: an instance of the Network of Terms can choose to make only specific sources findable.
+
+The Network of Terms publishes the following information for each terminology source:
+
+<table>
+    <thead>
+        <tr>
+            <th>Property</th>
+            <th>Explanation</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>ID</td>
+            <td>The unique identifier of the terminology source, e.g. "https://data.niod.nl/WO2_Thesaurus"</td>
+        </tr>
+        <tr>
+            <td>Name</td>
+            <td>The name of the terminology source, e.g. "Dutch World War Two Thesaurus"<td>
+        </tr>
+        <tr>
+            <td>Alternate name</td>
+            <td>The altername name of the terminology source, if any. E.g. "WW2 Thesaurus"<td>
```

Comment:
Should we say ‘often (but not always) an abbreviation/short form of the name’?

Reply:
👍🏻